package.json

```diff
@@ -216,6 +216,23 @@
           "ui-eholdings.settings.access-types.create-edit",
           "kb-ebsco.kb-credentials.access-types.item.delete"
         ]
+      },
+      {
+        "permissionName": "ui-eholdings.settings.custom-labels.view",
+        "displayName": "Settings (eholdings): Can view custom labels",
+        "visible": true,
+        "subPermissions": [
+          "settings.eholdings.enabled"
```

Comment:
We should use `ui-eholdings.settings.enabled` because `settings.eholdings.enabled` is not visible to users via UI